requirements/release.txt

```diff
@@ -0,0 +1,10 @@
+cached_property
```

Comment:
change these to be only the libraries needed by the spell book

these are left-over from merlin

it's ok if it's empty for now